### DISCUSSION TITLE
Who here is greatly helped by the copilot.

### DISCUSSION THREAD

**Discussion by yusupsupriyadi** [community/community#41883] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

**To be honest**, it helped me a lot, and I happened to be given the task of working on a ReactJS project, because I've been playing Laravel for a long time so I forgot how to **play Reactj**s. because there is a copilot I just give suggestions to the copilot and what code I want it to come out and according to expectations. **big thanks to the copilot**💖

**Comment by adlai** [comment]
What tools do you use for editing without causing excessively eager linters to make you abandon programming altogether?

**Comment by iamcjwilliams** [comment]
@yusupsupriyadi Thank you for the feedback on copilot!

**Comment by Urethramancer** [comment]
I was just writing an assembler, and it auto-filled a block of code that I thought looked wrong. But no, it had read my README and implemented a late-night change I documented in the spec :)
